packages/cozy-intent/readme.md

```diff
@@ -0,0 +1,122 @@
+# Cozy-intent
+
+## Concept
+
+This library allows a react-native and a react app running in a webview to communicate in unidirectional and/or bidirectional fashions. The library is following declarative paradigms and is fully strictly typed. An emphasis has been put on low configuration needs and ease of use.
+
+In a nutshell, one end says what it wants to an other end, and the other end handles it any way it wants. For more information on the underlying systems of this library, see https://github.com/alesgenova/post-me.
+
+## Installation
+
+The library needs to be installed both in the parent and the children application.
+
+- `yarn add cozy-webview-intent`
+
+## API
+
+### Nomenclature
+
+- `event`: simple function that sends an event to the listening messenger.
```

Comment:
`simple` may have multiple meanings and (_regarding which meaning it is intended to have_) it may not give very useful information. Also we tend to avoid formulations that suggest things are easy/simple.

So I propose to remove this word
```suggestion
- `event`: function that sends an event to the listening messenger.
```

Reply:
I agree, I will also mention that the event is just a string that has to be handled in whatever way on the other side